.vscode/settings.json

```diff
@@ -1,3 +1,15 @@
 // Place your settings in this file to overwrite default and user settings.
```

Comment:
No, it just should be two-step process. First, we check this in, so that everyone who gets the project can benefit from having these settings. Second, we add the file to gitignore. This way if you want to customize the file - you can, and no one will be hurt by this.